README.md

```diff
@@ -4,7 +4,8 @@ An extension for JupyterLab which allows for live-editing of LaTeX documents.
 
 ## Usage
 
-To use, right-click on a `.tex` document within JupyterLab, and select `Show LaTeX Preview`.
+To use, right-click on a `.tex` document within JupyterLab, and select `Show LaTeX Preview`:
```

Comment:
Could you say "right-click on an open `.tex` document" so that people know that it's supposed to be in the main view and not in the file browser?